gemfiles/.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_RETRY: "1"
```

Comment:
do we want this file?

Reply:
No, good idea... I think it stuck in there.  I'll drop it.  Good 👁️

Reply:
done